tests/test_integration.py

```diff
@@ -1249,6 +1256,83 @@ async def test_info_unsuccessful_start(succeed_immediately):
     assert "failed to start" in stderr.lower(), stderr
 
 
+@pytest.mark.asyncio
+async def test_filter_given_implementations_lang(
+    envsonschema,
+    lintsonschema,
+    fakejsimpl,
+):
+    stdout, stderr = await bowtie(
+        "filter-implementations",
+        "-i",
+        envsonschema,
+        "-i",
+        lintsonschema,
+        "-i",
+        fakejsimpl,
+        "--supports-language",
+        "python",
+    )
+    output = {
+        "bowtie-integration-tests/envsonschema",
```

Comment:
Use `tag("envsonschema")` (and everywhere else here) to save a bit of duplication.